yarn-project/sequencer-client/src/publisher/viem-tx-sender.ts

```diff
@@ -116,36 +116,8 @@ export class ViemTxSender implements L1PublisherTxSender {
   }
 
   /**
-   * Sends a tx to the unverified data emitter contract with unverified data. Returns once the tx has been mined.
-   * @param l2BlockNum - Number of the L2 block that owns this unverified data.
-   * @param l2BlockHash - The hash of the block corresponding to this data.
-   * @param unverifiedData - Data to publish.
-   * @returns The hash of the mined tx.
-   */
-  async sendEmitUnverifiedDataTx(
-    l2BlockNum: number,
-    l2BlockHash: Buffer,
-    unverifiedData: UnverifiedData,
-  ): Promise<string | undefined> {
-    const args = [
-      BigInt(l2BlockNum),
-      `0x${l2BlockHash.toString('hex')}`,
-      `0x${unverifiedData.toBuffer().toString('hex')}`,
-    ] as const;
-
-    const gas = await this.unverifiedDataEmitterContract.estimateGas.emitUnverifiedData(args, {
-      account: this.account,
-    });
-    const hash = await this.unverifiedDataEmitterContract.write.emitUnverifiedData(args, {
-      account: this.account,
-      gas,
-    });
-    return hash;
-  }
-
-  /**
-   * Sends a tx to the unverified data emitter contract with contract deployment data such as bytecode. Returns once the tx has been mined.
-   * @param l2BlockNum - Number of the L2 block that owns this unverified data.
+   * Sends a tx to the encrypted logs emitter contract with contract deployment data such as bytecode. Returns once the tx has been mined.
+   * @param l2BlockNum - Number of the L2 block that owns this encrypted logs.
```

Comment:
The comments above are incorrect since there is no encrypted logs emitter and this function doesn't deal with encrypted logs.